sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -1047,8 +1047,9 @@ export class JsonRpcProvider extends Provider {
   }
 
   async getCheckpointContents(
-    sequence_number: number,
+    sequence_number: number | CheckPointContentsDigest,
   ): Promise<CheckpointContents> {
+    //NOTE: This api takes in a digest in 0.22.0
```

Comment:
Could you also rebase and re-enable these tests: https://github.com/MystenLabs/sui/blob/ba4f235cbd2bb4eaa27a34f8673994c63fe7a115/sdk/typescript/test/e2e/checkpoint.test.ts#L51-L62 ?

Reply:
Also could you run `pnpm changeset` to generate a patch change?